homeassistant/components/zwave_js/api.py

```diff
@@ -465,6 +489,56 @@ def device_registered(device: DeviceEntry) -> None:
     )
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/grant_security_classes",
+        vol.Required(ENTRY_ID): str,
+        vol.Required(INCLUSION_GRANT): {
+            "securityClasses": [int],
```

Comment:
Do we have other parameters that aren't snake_case? Maybe make a code comment why we have that here if we keep it?

Reply:
didn't think about that, maybe we should just use snake case and convert in the command